README.md

```diff
@@ -1,15 +1,43 @@
 # Nibble 2
-App for buying foods &amp; drinks at the office
+
+Application for buying foods &amp; drinks at the office
+
+
 ![Nibble shop](http://i.imgur.com/1ItEFrd.png "Main shop view")
 
 ## Requirements
-- [Node 6](https://nodejs.org/en/)
+- [Node.js >= 6](https://nodejs.org/en/)
 - A server running the REST API [backend](https://github.com/dotKom/onlineweb4/tree/develop/apps/shop) (not needed for dummy testing)
 
 ## Setup
-- start bash or cmd and cd to nibble2
-- run: `npm install`
-- copy and paste constants.example.js to constants.js and fill in fields in app/src/common/constants.js
-- run: `npm run dev` to build and run webpack webserver
-- App is hosted on port 8080
+- Open a terminal
+- Run `cd nibble2` (or where you cloned the repository to)
+- Run `npm install`/`yarn` to install dependencies
```

Comment:
This project already has a `yarn.lock` in the root. Without being an expert on package managers, I think suggesting developers use either `npm` or `yarn` might make people install new packages using `npm`, which would add a `package-lock.json` in addition to the already existing lockfile. I think maintaining two lockfiles is a bit of an overhead, so I would suggest standardizing on yarn, removing the references to `npm` and sticking with `yarn` in the README.